src/geometry/manifold/ManifoldGeometry.cc

```diff
@@ -168,7 +168,9 @@ std::shared_ptr<PolySet> ManifoldGeometry::toPolySet() const {
       const auto id = mesh.runOriginalID[run];
       const auto end = mesh.runIndex[run + 1];
       const size_t numTri = (end - start) / 3;
-      assert(numTri > 0);
+      if (numTri == 0) {
```

Comment:
Did you manage to reproduce this? I think this is the correct fix, but it's still a bit puzzling why it didn't break anywhere except on my machine (unless everyone else builds without assertions..)

Reply:
I just reproduced it w/ a different model (while testing [rendering of modifiers](https://ochafik.com/openscad2/#H4sIAAAAAAAAE+V97ZLbOJLgq2RrZ2bFKuqDkqrclpv2lsvuHu/4K2z3zM5K2hqKhCTYFMghqaqSux1x/y7uJS76Fe7v/bl36Se5yEwABPXhLs/2zF3EOma6SDCRSCQSiUQikfqhlUdFtC5b4x9aUVzJa/E6qlatcauXp9F2WWQblXTLOEpafmshompTiLI1nrTWkZKLLKXiqKw6cbls+a00+rjtbJTMVMtvRWUp13nLbxVCJaLoxFmaFWX9vs4SuZCiKFszv3UdFUTDbxaqNT4f+a0/RoWM5qlojatiI/zW21yqeXbbGgd+6+VmPcd6+Pw8motUJFfXUbpB0oK+30Icg77f+qOIq6wYtcaTYOAPR/7o3B+MZqY8oHL7OjBgtmToVJz5rUTMN8vWeBGlpfjkt8psU8TEjB9a+VGexZmqhKpa49ZChYP+A+j1YDIcj/qzqZqqRC4WohAqFm0PfpgqAIDfxpu5aAdnPsRCVaIIsf/eA/5Y5itRiHbQ9+E3CxUuFH741Prk1zRkuVDYeE/cRus8FWXvcVTKuOxdvv1un6q/nYK7tHyRXEcqFknvu6ePD3LkzDBkyAzpncDkSZHlkGQ3CubZ7XgGJ72p6vUgztbzDItgkRWgSASmSotCOGBEfR8CHwY+DBndbrWyKqRaTtVb+luG09Yiy6YtrrzIMh/mUYH/+ajrpyxeB1svp6opfWGgBzjoj5/7MOiPX/gw7I//7flnsRmiGsjCaeutIezt+O06SlMfXoxfiERu1j48Hz+PiqXQTJvA21QmooCZZlbJrzcyWYqqwTD9JRyOLKlBXzO/V1YiN3UblSqRcwvM6El/fFZXO4HJ5UrEH+bZLY8WokpEGRcyr2SmpspMZggBZemBplrP6boSlFwCN7JaAVMjPwoIpkrDQghnD8A0+07cVk0EWHJktMNpayXSFIfb1H+bi1hGKVzTfDei1sDBn5igVJQlVKtIQVaA+OsmSqHKYAQiFWuhqnKqtGYJUbFoRg3GZ8gnrWRCrWMOfRyGtbo59H0UNvRYE4Q79HuZJEJxR0hfvciSBtf19DXK486aA7X1JhXl/izu9WD3O3SAasKmjJYCsgXorz7IhQ+0EKACK3u/WUR64C5XkVoKqFayRKaSlsWHQqyzaywXsBJpLgpYimwtqmKre9iQqR58l2bzKIVClFm6Yen7zaKEEPrd4AGgyD/J1D9XsBRKFFEloMSphfIexaLSw9vvBrBeY82IBQ4O1UxxBhYQqWUqdMUzSMSyEKLU1LyIpHIJPqBtJarYUsRIa12K/+ZZsm0bvdsAzQpb/on/rLJUlFT4STf9e+JWOVVyAW3ilKc5yHDM9awQhj4ZQ17ItUQjoOzybFqJUkAcKZgL0Ot1gqMSFwI5cI3zGgkvcZBxjBZSRSlk8/cirrANHnjdFdM5EoD2tPU43Yhpy9tZU5xqjQ7v1n4jEqx8bLFy8SB3XhFN+2ieyzUREW9TiXZJexUO+j4U4RGUvR6gOss2pZFq1hOy3ESp/CjVkslei0Qij+JsnWeK1cN+x0Tya0rCbp8v2h4UWRVVoj3p+/f7fn/mNZjxoAH+2AFH4F8Av2zvfW18L+tOkE3Y7JOm70Gz5PFeyeXRDhphNlh7PXimlCj0uEApVSxAVv9cQqbSLSghEpGAVKVMbHWuqmukUuHMrOUzij80ZKMIAx9WIdpfO7KhraM4SkW73z1rdLUqIlWmxNbOEPnaGSFnHYjGmKJMPGh+czEEZ4hheDbz9uViB9YBPSQxh7Df61Kd4F4Xa9XCM2RpYA4dr3+seudY/U8H2XScS7WiO8qfvu30voztMqgJ+/hzsMMm7OXnYI8wYXA3Jnb+U0MQHGl8dPYlI0C0+p3BYAfNMSxO1SM1DxJg5rSIV1n7WhSlzFSo/7Y9hOr14E+FrCqhYL6FF1EhNyX8AcU5hW/W9PovH+i1q0T1ECvw2pXRgiRu0U6BPCtLiRboBucxpNGNT5+jTbXKinbpwSq6FpCIRMZRJRKI0lTvH/JtIZerCiKVQCFS/qoSUEIuV/MMTUsgCDJcyIAps0V1ExWCCwQhyjfzVMaQZGu0CW6yIk1uZCK68K5RQ5aQSDRc5xtsCO3ObIONbwnLTVQUkaq2XdvPP2cbKFfZJk24C4WIhbxGGol2szBfXvbhNZPwhEigyk+4vzJTEKWZWrKd2+gDWwLPFqCyyodSCPhmVVX5uNcjI0BeizhbrzNVdrNi2eNecid7H0WR9YJuv/ewO1UNU/PbjaJlLkpf5UK9vbx40ovSShSKEF6xQXDA2NyUAr5Z2MoE8HCqqLjczBN5LVFyTDmv1ye8aNMCDSWZtgWy5Vov49waRHleZLdyrQ2aRZGtQePEZSNPoypTMoYyS2VSMvu0PVdZG0qUgKNY5lHMltJcQJFtlqt0C6VcyzQqQCoyClJxK2kgF1nRVuFkMA7QlNcarzGfTpQPfZ/UYZ6l22HS1iZTEd730ZZXnsfGrWFnR/x1I6+jFGn7/o+6i0fwwh7eQRNxr1fjYAm5FkUlbiHCiZUKWkTRzFS/HYRh39tvZHSolSvmLrWVyOsyjIVM26o38HwCDN89fffm4vdPn7x59bLteawyuJnR4Wa+/pJmRqaZy+8fP70z/r1BIJv4iqxQ3cJVnG1UVTdzB8zB4G+i/NXlu8P8OTvcyvmXtHJmWnl2+ert4WaC/sF2Bv0vaSfom4aevHry9LLZ0ieawD09ld2ZjH6B6yzd6K1LFMebIoq30K4KSVsyEIuFjKVQ8dYjEz2xs1WLcrXNzU7nchUVFVzcihImtz5sZxDCxGDiAfUh6EMPRFFkxYwqAbTpDUIIoEN9uGKSoAdRXG2iVL97SP4fIYRRb3jy+hlNqQsCQNPYEsTAU/USMfb7Zt+5jm4djQG4e+ce6WpxlMablPWWb9WNKCtUZiJhV06moKwilURFouuVU7VQ2NI50RMTCxJZohcTYlnEqWjso99ACIOzz8BGidzQLocN6KA3OJkEfuAHMyK3nYcTreuG42D8Em0BUbVp6K0+C9izqTwPxWyiTtqDE6sXOgPvdHDSVh1UEUGnlEuF/jLiGuHxen+czbwdPZd3b3Hsenl3O/M0se03rgeV6R18Mb2DXyR4hASrX5XYmr0BkstGYZPcAMntDfYJrudg4ANNQnVY11JfUqGo8iSY/br8rrswPJn0/eBIF4iNL3vD7pnndkVlxZrmv1W7gaN2zUAMTv5u5Btx+VuI318z/p8QP/zbif+MEJmV9B8lPQPsQ//AfMU+nLmUZ3EVrURSZOoqSxMmvZ60f3dJ+SyhhtlfxOrm0v8PnK7BQaHpj4PxyKVfxlm5y/F/GMOHxwl1GD76QunesYP+MTy3E/WzsvP1F3Vk1876+/aktt+0T1naLRc+Sd6PoVMZzyrmmwpPoJarKt06FpvxodO2jPAkYhFt0srueKCstqnoQht3u7QrQZuJtuAR5JlUlTdVZusIjfU78SAk7qGFk0AYkpdgAY+ggDEkvQF1mj0ZV+K2KjYJcySMirhdhG98PgQIg6/7frZYlKK6oPfO/b7Hg0kkv25uIY0PUW/z2RJkI8onD4OxPzvkvsQDCrRWbR8aSzbaqzSOMOkEfgflxZ/QQwefsICf2MCBWa/8a1G1h56va6GMDej7kP4O/KHfn/mTIU0jmMHsgdM2a3qnUTx3xf9xW85jRz+CfWTPj/7Wqb+ZR6hp5sd9Yg2RfX+oOzrwR/R35N+j8sA/p/d7+Fe3OPLPqHTknxPU0D/3zwj63B9S+ZDeR/7AH+512dW4TsfJ2cU09IlhvKoS+aagE+iHPrG+ZvmQqDas71O7+Hfk13w68wfUxzPd5zN/RFBnfuCP9qhsKCkkkwQbj7ECH+YQQjs4JWaeeV5v4AOeqKJB1Z5E/nymdQFMOpE/J5rrvx1+6Jgn6J3AMksTodDnVPGeSyr4tz+jr0QJOoTkkZ77fT+a+RP3bwcfOubpKLJ/30WG8P6cuFT/7fAD0TaDY8j+XCODGfTg1gwEOkHu+4BjASN6/Bofv6ZHlAg4o0cjoPdxe2vGBwZ+EGDEgT+BIT2ibME9fAyQY0GfSlHy4JweB3Xlr33EjeOKYQtnFgwfA/1Y48HG65axoG+bw8eRfhxZevDRdIbiInYFprkYaEUY7QjKyId52B66BUhD3jcTAQemi+R0z3w8sMLhqN9peAALOg6AW4AQhiUw98FURjDz3jHvnV2ATgPC0sMt0Seihxsy751dgM4OhKUnQjD+DzVn3zv8rp9rgLoAIQyezg6ezi6ezi6ejosH0TCyBbEdDN5gAD7gOAcjH3CQUUQmEJwDrsw+4NAHthgQOvjaB5Kscy62qO7pOvQB/0/CFdzHFoY+kHa+Z4rhXKNCmQ7uc7FFdabr4AcEgPuEaqSJJQE/s8WBoWqg+3HfGYCvDVX3NAkk3HBm+j3QneTiwDR8plsZOrzLA+QdWlALCGHR9+C63KzbVJJLLPIg709yOYOZZ+qg9VCaenkYZyqOqnbe9yEPPI8W7nbuMTBFsaSyglyoKlqim5qXbjyCy+yazqfGjs+aBpbgDH2SzOpAW3aLcNGfLOTM0+POBCMI2t2o5lU773unC+nDAunHP+1cngbeb89mwCyYKg8m3B2fW5vV5ol1dvMXn/tR4fmGSJamA3jUgmEPHBuEfnTTA/aJS5VvKt2T9aas0ANed9p6jyV+zxSdTTgusD0TzVqxfsJew4CNWNRU2CVkDTPkmPHm45BkOZ8YaPsKyB0YrUWF4U1YO8Ha8rok6HKVFRjPkfC3/BpCanRCy17PCATHBjFMZWEChrFuSDw5vm3AKnHTECloVxjFcq1NeCzNryfVpD+b+UBPgX0azLQNrmUA2iwm42Tmw3v92EExQRetPH3/VdiH3/0O5Fdhgn/ffxUmHvXxg8whK+SSIjbirMCTcyarPtVDesrN/DpkMq+pLeNX+hAmHdl574OU4UT67/0PZsqaf++DUMpJ+5qEcDjDLbeUk2s5O30f7IDm/TDHLzilwtytkw/0+4Ded6pdybBMRZHjbMwDX/US9CZfvXdKB1Tq1vNAf72Sfn713n8f9JQDMHOeaW6TikA+ePqLIeMd+k+NxLaVB+qkrU4DT0sdy8AGD31syCBVQ2l7p9rJaeAxYFZFqRErRzHgwRNOJS03KBrvMLxqSLWUuDF1MDDKDmZdI81pQVft/NrTJEdpaoXPKLJrv5ZJA4exVPJaqBqtlWgCRLmmSVz5YGvgrsWUS//9blfmUSnNtCl9+AEkhA/JY48zC//Q5vP9bilK3CfbSJwp1CC0s5uTe9ohSdyiSipEuUlJcy02qWYslKLq6gkt3zfGza986b/3QJ6+D8M+PAKcezAGGYYJvQX49t68DWYwNhKS5nAK6kThHxoaHtaO9OAU3kMH5yehDGAMA8Nco+gXaYQn222eXihMNIFxxKprr2N0P6uHiUJFf3Du4waknv0Bzf8JdtP2za/fToPmO4D//jTwoJ5bBnfwxbgJk/PeCTRuLSMzPbk8SOAhBPAIJm9OrEiaNQnGWGw0rtaqZqXik6Bf8d9UwckJPMfgTrFYCIqC9yFD0RIYIKiPh+vz6RI+iJzDaguhA/pgLtLspgsa3a/3r8dDgr22S+PnPeDBwLceDeT10YURP/LsDvXelIo43CEEs6PFsg2tSnpriwWJvB7wksmtanQ4flf14mZNFWi/N8Jtq8x8nP+7hZ6zpLv4INS0wikMun04gTYTegJt2UFycMZtcnzHt857z4NeTSBj9ODNCan0GrFXG4cplmqN6fTEa1qBWmvWM1mD6emzE0FH4hvuYavhOncFRO/SnQD7PuG8C+AROJqidkB3h3LBo3Y2I1W9M4CkLvbH1egysEMr++HihDh++v6kbQFPA+9UWk0kg1D2T2tjQQ7wvQFdfxuGcnAamIGeyL4vA18OfTnYt34bxm/T5tw5ndmZIX/jPPv/aVLlXzqXoAM7s8kyPR+Qudq9PUFPcN7dnuTdrZ93P57k3Y/1ilKgnynv3sIJkA8h6Pahg/t3bCIfdLdwCvmg+9Hjv9sTfOkNu/16eIstodgeQ/GRq95qFB8Rxe0Oio+E4uMxFLe6dY3iFlFsCUWtPSbFrV9s/eLj7O+kN+A+7tlJSOAuuuOusKg+7gqLcFqH3AkWjsH+l1IkuKui4wxngYb65NHRMweOIxNfYSxNeMdNrTZkd5QCwJ3d0L7xXDd3NDvDRA7Xkfaxj/TBAPufh/7I79cznBo80774M32AcEZwff9spxWUE3Q8s2ncZCWM6y3DFRW1lT26aoKyeJETl10z9T7dLvTol4GGCYk7Jwhd3wvICq90UNxfu3ZsGF4MUEOO/gO3iPXmP85UWRWbmAJsMFZw1+NcQlSiJ+FGARmQGDVZYtikDpe8kR8kXRKgwEl8630rUzF+Vp+cdhhnp8bZLa+XjhwdOmU1gnS3dSlfyb1DASxfR0slqw3dHuLzgcDPV3LGs4TOE3oWhsrwcCFfyZ3SXQm928lC77MnCwbTXY4VjmD69wamO50pHMb0ZwfT7JCiQyOLTnlG/n0fPfFf+/dpXpz59/W8ul/PowkdJvD0CgI6pdo9SMADAjpGqE8RvvbP9NQ805/PCPW579T09UNfY+0TVfec2czU4RmU4y/91Waqx5MUjs5SZ5L+UrAwsfzu8cFTxRWuErHOmhdVGh849DZiMw358sAINmmzEb/H+H6ftBq95yn6lAnRt0W2fk39bqN4xxwsiXGLQKpAfyPZj/08DSb92Qn+CWYzhqVA+DwN+I1jAPjM1u930Q9CaLCtGorP13mslkiLPoYvwhEfxH/tTfqaVgQZWL+jGYpl4EGVDRMcB4IzsCuy7qiSDxNZ4CW9dt83wQLLgdcJvJnhw+AAH+is1iw3fT6ApnVH86anezwwt2jKfI5kkfP5NbaRqcdb0+MBS9fKh77LoWmr4NtgfAGHYz+6/TMdg7rMjIuFMc9zkr0qG1CPm5jm5lbaXVENj6A6Omy2q9T1aQuxTFs+/AKJn3Y2K/sSh2dZ8b5nPBqi/wiHN/J8mNu3uedDbN9iYwDh4kiTMi6ysmzPh9CBaEig+OAZuBvUCyfRsF6ElX/DS+w+jd9GMcdM4B3QKBY+pFFZyXCwT65dMowDqLZMah/7hLBYe4KOdfeFj1FNEJS86e57sPNO9Mwcm4KQubrCHcNICZ8uJ4eToN/38f9aSn6ouUhgZuoxw7goMEVmC4g2tC1EBRTFWdlWcAKb3NyVwRMGHhLlO8V6Bk1bakNSpPz6m7bRoxAPEa/D60asNk7Im9rXTRte7NH+vUd2PznDupDoDbtSkWpfe1ab4B7vmg8ebsPw1oPb2mfX66FXDJ69vHz15s3Ty3fw7MXr509fPH357uLds1cvfXj59OmTt/Dk6ePvv/vu2cvvgBxzGCZdVZVY52hm5VlRoV2Gy8C/UsxPD+Jy2X1fAk6iPX0xbVkfL6EqRLUpVAmTRZFhPHec4WhExbfN18dR/AFzAMQfOMxbRPEKzb0qkgpdyhGksiRCjNcS46ujv7mTjvfhkMojSdNKj4R+f8aIHKdLIDpnej5cvnr9/OLlBVp+fV307ZtXL99RGAi/P764/APatvr17euLly+RpBCGuuhlhHHo/Z7B4J4cED0H5+Zqf3I2z7r4lJX6YTHyeau/P3P8xqTRAKk51GjsR7Hx15/XA7xFpcegfhzMrOZ7zdaORVX3kO4K2D7obWlKm1EzgXA8oCKlqDHLGXTgBuwUa1fwDXREXnrwiAdgjGUPQRfxGI3t8DU3Tbi9FFERr6wgGzlMhVpWK+ijIAZ4TU4Hx9WMWWbq3TZH1nAbJ8RARtfmMl/30UPvh6HY/UcENytiUV3PCANOJ2NYmHbDWsIeWeyOGmmehDT4Cw41rhsL/6HboC3hFAIPfgtp43SwQixE2wTPFav39vV98xjxWhK1esh8uH5fv7+f1ZAeVPIU0RzsDFni2OhXIbPqEUgY4yzy4Vqetq/fd66ld9Ju33TUybX0em11ogs9PCAZNxA18FxrRJaWmefhGYnpBuqq/9oMZ6muOfW3s3wX0zGeo9fPuPxQ4D0f0rkuwPGo7Yj2zrhQC57tiI3HILLNf3Bzxd93q2uZsH6XRjWj1Jy+7da3a8OjuvdtdcK67yH0dznsYD1E3h4VDmhjKN0K5EY5LKyu8E3SBTwMYQiPtFIZw05v07kBoDnQHDHjtd3dUM6LSKapoJ2kfj6U1+RX+EcmwXM5J8cOuG3pC2SXnEhDf4Ekq8ouwDs00uJCYugMJqPBe6QaggJokMoiIqNB318DuNwUGCmdbuGpWqayXMF3RZQIGNiq7aePL556FHS9ydGWEkkXuU15E+J0k4hyrJFJfodvHr96+3zQK6vEbKWbn11OCtXZlJ3l4JchXT4wIPLnuyLb5GN4zB9t8dvNeh0VW/uBeKTDo5lLuTHwfp0Ro5bfchqOutVX16K4luKGWn5dSFVRaiHqpIGRJURwU0gKPCi3ZSXWsCnpfngkS5Ew6dUqqkxOlUJEiblzTclTlhQjQljnW8hFlpsxl+s8khgrrq84A7yIlBsin2KoSYRxWhr5jb42TyQYpJr1ABcwbRnJiEWaTltIP5Ix59w9GJSlL49zX6hSowrjRE+oLCvKARPBEG5hAHRNfUlpkShETMv1ioYPb+NL9rbSlfdEqIpTy8y3GqcOlxlrMVhuCtEePPFfiMR/meE1TW8MF5jhA91wLk2aWVXJzDYZt+Zb7Ih04nCoGXSOBOcPmsxpT4LBcHR2PvPXolplSThtIa5py3tge2xcBZwgZFnI5CrOcinKNl79xmxT3f79wcmzl5e/x6gndB/NPF0bV0pxW+HF2KpoDzu/KbKb09/EWXoy9HzaUnbvnfmRildZEV4+ffnu6Rs/wnU0nLa2vDWztFxmKIibeua+UgJuoi2nTlrwxYqo5tAqQr0hCuA0S2RAWmapzVoUMo7SdNsFeKY4EUAq50VUmIHZZhuSryRjQZ5vYSmveVvE+bYglR8ETFuD0dm0hRZpZGIF6UMwGJ37cLOS8QpKzL+1kMIMB5fyBC8E5IUoBYX1kBRa4rUkaayYZYCDjqIKne4KcLu4daRBZSyAupWuw6o1HXuK21zEFfaAj2/WUZ5jT+aiuhFCsRKWCnd+epJxX4ympUhBwknzZ5fZKPqPNxVPJVltOOGCxPjIeAVrzMGkr/d2a2VSRR8sXzY5Np3RVKLUAlBmOHzr6AOvEKwQ51n2oTQJrTi/BcVKKT5zWhzSBDBtuavJtAU3UQmJuBZpltMCASYNlAMGhchTpAOTRogCNE4WgJK3JBX23AgDttuymk4WgMci5XY9z1IfhqNzO8ZRSQnDdA2zPvGAb/hbgxBM4mDiSu3gIDPW0RaVIDFW5/SYb6HYpJar0Rwza9yshAJxG8W4dDZQx5zoLY6UyiiMdVOKZFeI3jlThMfMRcLt8flQVN+/Nrw39XDKiQRf02wjLX0qwSZRKWel0KiqDPIiSzaxMONLbfEt7zW0p60lL/vowswW8P1bK6W6VV+jR2sA2uYjiZ2AC9YACp4mpKE9VAQVMVOgIG1QOSAvbEIV1C9JAlm1wlBawlyvRFkB3z99DO3vFev3Rmup8Ha5aRbWhkHEgxvBt09eYIYypdVXvEKuqaVUy3ppYcVxLYotzwMjmBlP31VWyI/oyUkhjbbZBjOakMxim5VQSalzdkRQimtBOiibp2KtL2SZRvT8MNNTJaKUBWU4LFdRkUMlc5z0tjsc8Uy8i/JcRIVphzSAiEpJWu2ZAiGRkTQPzEBhcLQQ6xI2CmdTumXNe/HkQkt3hIvspsK4a1nWI0Nss0wjNhlfnDbLDpl35tMBc45knW/pbeEiwXBQSsgTqQy+ia7Xo3+hMOI07Ypk81BbUv+arfDIjO1Nphx3nIYmjAE33IFyU+jQc9JK6MDDnuPUFSVn4EFHHwWKtzDBTYoToxRxphUw5Q41iUqiEl79gSrF25Re5xFbvGxVZAXFX2qZSpjSRXSdoZXltEN8nLbiLdlISrcHpABrdG8oDw6mtk0YI8abYg95aTaCiEKJ0bqSYs9xBRQJnK3XvdJOnq0oYCVoYeoGupRuHdJR9Y1MqhV0R4yUPpuT5Zubm66eOJzXCPPL4OEyGhWdSCUdbZx0NFSnXqJ2UGESQ4WiLpeqG2drY7PTtdBoKUzTT7KKU3Si8/s+BdP0z4dMZG3u4L9+tz84Y4DhPd0/AxGGD7HYZpBAA5wppas4jJdZQas6XhbFv7o70O8ORkGn3x32+4YtV5rgK8zOof/hlaSzexibja2Ru92CaYaHGoxusxwAS7LqqhTotCew+wMNdv+sAYbUabhJvzs4GzHY4F4DCjNh1VAEg1Aj4zXXxw5u4219P5TJ9enOqzdVdOaAfncqCMNaevUeuJlb0OZUc864GGOP8c88m5CDC+pUX5iZt93tB168mWcyaU+GJ5om+6BH9zSYGSv28at3jYSVDVJ5phuiKJ0KIzoJuoGuJQuKWYOOxq2La2K2absIwqzwi0EoC38Vag7Fq2i9EMUg7Pa/5pdILQfhqL9H2gHCcMZ7h1oKbU9NO0BpnyXi7g52cH86tql8oRNIoo4yp9ylPS/hr9Zt0Ob9w9utyvJSluNDvgNSrRHbq9oaZ5vL3SrzXWtrN5gtocH+LlqWY/hO6HvW32MOV7O7MKRgAz5MaLMywwdiGT7pLRM+aibgY1ZIoSoGzaUK8Zjs4t27i8vfP3357u2nBzoZmSXRtPeEM8zgvrrIbtCeOWhc4+JPEp4XooqMtYur9V+Q0r9AVCw3tBdNRE6rvL55ZDYgCfyFCf+LsUSe6fq6GBdWsw3ED8pgpj2/8cPrzSUKqjsEzT0nLYFUw5oT2UIfMjV3OKWwGCmZHBJsTEi7V9NYC6F3S9x9lANj76GJWwgIahje51dZDqlYVEiDD0Peu2RVla2d4pEF5QhR3L7X251zt5IF6BL7mqzbKImWg+FeRSOGFou5IxaBBmlKL5HeZI4xi0pjLEV6U0Syf95BEo2YGJz1xpds8e9PB1/3+zgR8Gn4bRfgW7zjgsfTlm7tRkHaO0J1loNpy5pkSu9Vab+iRUFn4cO+KGvp2j2J0yuz/b2RacrX44y7L+E7dvXsNLa6xvb9W1ge9Op1AV5mlWDhoryHrvm2O//JUjWM1wZfAidzscgKcbLjKiIEKE+I1u50Mkxh+KFrtmdm2lxw6+sMZxtuQQshODND6c5eHCJtsVK6PMcfiXN5mUWp9lBo59CWBQ2twEwJzNfI6RhtcuhClJUZnTKD6DqSKRvk3DjuYGLKMV3vKbXL1IAgJfXCWdugvjY8aTnoArzRxmy65c2jNWXZ1/ftkxeO/Wp8akj5IjOyu2fxUhN1Vgy3Cri7FgNfd8JynrQtpR8uOU+lvDYJKGEdFUtZZ2kmWZynGZ5z496WMmFyOjK+6anhrNsOfRUiUqVx/+g2jR9P+yCqbCloQtYEp1ugJOr19kdvo4zpVk+aOCswss707EKrbOvZIqJDrTWMW1ynIPONKjg47fyGavFZTzg6mxUpCSUn8tfLhN6e7C4NnU7HCBpaviH/caSbJdvSNG3xVpScYdMWpQ0nQXrCp7TjGkLj5QYhNA+ONq97nCntKTFTnWXWKlqfm+M1q9laXcM6braUnZ//NppjEXBJ1RPEsIOXebyZaEJLaAJyqb6XV4JWS/raQZuCwTwX8V/Y0/kXQ1EuMQ7gDQWwsCNyTf5uzq8OEdlbhPPfIbrFFhYVZWPHdl3EfY2RbRAIgbPpk4dUY89yqLKbqMCcLIwGm29Q9/1rpuypzo3fdl3RO9bRtPXzT//n55/+988//S/4+af/+fNP/+Pnn/7bzz/992mr9ig7w4RO6OHAO+KF9vGPb57PfOObZpPL6qgdVzUl7xwcQKr9sv60Fei//ELSEwxH/nB0ju9cej5tOS2h/vNhtyU40tS0RXqDDKRKlFV3D9by8rFcHuclLgg0LWE0osMrnbCfEu4Hw8E+Xrf28yz7ACn+5/mrV3/A/3z4LPzv8Sci4E+YnvcreIa5WTkhbHeXp5r4hhxgGkmOEbKqsMhuGiuf/iGJRdWJUrlUvEw6Vg7BYRJyzqHP7ItXMk2MUsZ5zyt1F+Ap1kHTGDmNHi4nySXnQzLrBBNvTzD05g0Tsw4xB8kM9196v764SdpDj8y59qg+tTDavv3u1etTPIU+ff7023dezbtXJIn13ufdqxcOnOMOsJi+/dOTXSTvbjIcoB3xBoP1i9GhBdJAyFK8h89w5tPx0b2oqihe0XnezqjSIQsFQdEZGIbSmZVOodnBQkG1+ACXTnHgMyLwBTJA04y1m937GMLINinxIAPby5TrRHWMCCMO9/qcfWM4cuVhk7eDM5fhETGizYEMDTePw3jcXcgjEtGs5I6fM3QvZJLopPwOkrvWfh6V1Vc79R5wjXqkzc9TuNtaq9+cdZU2unaVthrR2orciM+D4OPiYZwyqVhG6ZW2MPFSgBZAK9k1Fg7/5ApMhK5hFO3RpZ5rJpv1eoupYaKyFEXVluoKjZw24/KbmD3yfLenLW1imN2AtSr8Jngjf8JuC9Q93+2qRs8fpi22K461oauwG8ZIrjF+QrvS4N17cVu5hIwdoHqH+cggudJlNLBes6IBqTdfDFWzMrT95Ea+qin58UeQJfedKuFokEla72Y/7xPQemA61Sin05buivU4HeY0xudOakm05iTGMLbIEDUkIE4Gm2rTc2qgp7jq60TOKtEWbFgLODyCPoyBnW882XEv1dZzqSHtoSP0If5nJ0OHDpKmsCh96nOy5yedSJXM9Gn5nnv04Ed2u9WfZuw/JDfdUSCdyQ9Dhdq7Lt0ayuNpuKgOw/T4WP+07XapExzrlQbXg4q31HTShmb1Og4bXZC6HANIf/c7qN9QBPt4smcSuMgFn3/jD1LRqbf2m1AYgnAkiVdz+TnWN+QOCY1XIU7iunnP2/k1Di2dTkhovCJrEm3UacvzHvaNnvkuwyP06yiVCSRyKTFtCMmhH69w+uCLjzZMY+eBOHZ+6gIjElWYFUk7Xnkd/Evm6o6GcgLdj4/0yWSRZphMszf0/I767XC2gwSg6X2vTxXuJJTaP9/oALmZaTEvhGpb1/Mn6/Y1juDxnhbbcwDrRChmd9uwRhpuYdQdTf/uE9Q6x/y7V1Qp3COgrArvqK9WU1PWgSCOHWJIbLPnC70xJjTs4u3ls2fag1d7dc0aT/6wWpu6DiStWa0FZE9Ay40kF5dGwbnfOVU5ad0ffrAm1qdP2muDVpVV76yJjXmkbUHTzyO8Ob7p0z/N8JqP7UswGzsY0H8D/E/fByxk6wuNsMFwhs3buwGHhsJkCuN56oH5eIVsxNmxcxnHfscu4He+bRyv+HJfH1/GzZsF6KuKUpuTTIaTs3EnGPd1fimePvGqN/gP6f12gBl0AifUN0eD11aVyW0bkXkeBP3/kLP6ok12U57gl3qiHKbb9IvpZsqnLdALFnGZfomQhtyRK9sxrT02a6s/gv5gZK73a22mNuuHnDRLbdbfhOdDf9p6mSlzaArtc5QB75BvGX0SdgFvsltt1mR/2nn+Fm+z2ClLvkqpAH830bieCvypmvnW+pdMGJIOYpmqq5IckVb/VPpHFkuR4i9yNYp3l+X9f4gMx4iGS6iq2IYNDB50UNHTF0zUNKtz255M1Vwsb7IiGdvQIAwOE0uplFYF+BWzGtvgmLGZsjpWSC5VfTjAlp5QyR5SoZIa3ftMqiaENHyiYMysQO9ZmmY3HP9zs5KVYF2D+7NIt+3vupMJzggPGU1InkOH2vKOrVoJE9PBxx7VDbnGE3QiLqTCH+IpwWIoNTW806OgGd9tl0nTcWdTlWY3v9S7HH/wh37YDLtwpK9t9PLmOJ84+MybqjV61ne4KxWsedOF9oTL57U8DmphKOVWMlV5VFQ7w2bYxT99lEDbkt1zSebR8KYqLxa/1PGscPyKmthIk1Juvqx6Q1Atkm1KRwjj+uDM8Z5v9c9LRTEfiuV4naJQGA0pFLrVG8Fk030pLTcF+TAPCWYPXaf1cE2V/QmUHsAl4EolYvKtGhcuF6Jd1ShJZXWgMLsRBZ5cNUrXUbVqFCjMf0XxHgsKOYUOsXHFsVeFljVa7zkOl0ev/s0qlmYfEvrdTIzeM/MNEK3TlNtXlwQCcws2eV6TruM8rhhnnEZlST7sok2lppc+vtmK5rcv69VlXlzhtgE/6mWVGyyr4gr1S7uxyOICIlUS1vautaijUmtKvLqLP7mXhOFkBo8gXpkNZwMKL8qpBC+mzrzdTERIldqsr/hqF9LlI76+Xfto48bXTYgus35TTVnSqOMXskbxc3B6GOdp4HFFxPiNRdgJcAU0CHCd7U5bWHQA/2lALQyOtTDQLfR37BHkMINeSXVFA0h1gB5x5xAVVdj3QUOFfU/3Xn8Kbfd//NG9mIZ0EcTMJ1SexyaOxkM4xnds/zSw7Z8GB8cJg7OiuLpCaW3zYksC/jJai72MLnbLzZRODCBemOKqeDIy6dfmkbPJkirxwpAHjGVICxDfKG5SZsjCycoSZ4ij7j2XJZ6F0RIcTjDXzq58MXv12v9IgxpZ0d4PQ8SMLldaxJ5zu2j3ChkAmRD0C0t19WDmO2+DWX13qN6T3a1LE15bzSEctsW9s5J+HE+NRtfnag+OGE9TxbqHhh7CfWlommCoe1mlsvuFdzB3r6w267pyHOWlVn13qozwDM7152KJRbQY3qW+A88IhEq+CIEDzwia06jeedYmPl2pdbTtJF7Nan1rBw+zLOCUYVHZ/24kbK/NK1nyrOcJT+2iZy8RCyLid7+Dpm9Da5OH/Qd7iGgFZ9IbqPGZF8J45e1Xc5a+I5UdiMMoeFSP1HYWx8O1cTzmhYg+NBDUvfnxx8OU7mPiNeEIGdoQ0TTQCmtsGDY+4hXG8esDk4Lt2gTP4tl+AfppELkAg0lH2+eFpEBXDJOmgBqFgdtACPeG2+nCVRlHSgeySZXgT6QUVvklsghDWgBJDeLSQb9IzuqDvnbqz6Ql0Q3cBHMZiZ/tSrwHowfQAbJMZ542qmNahrr28S6dJrLgbu3I/UKqhCfGFbHJ6FCHGVU0by4HuqC5Ikya07amkn6ogLo5Y0opsTVbyCFPSUaIs1Lr96/KzRwXZE0S0kKk6GrY3jHKLTAjxYSz/Lt5VbyCBZpdlMnbFpWbmDcfaKffQGx9BgX+bCaGf9M90NuqJr5eALIcN/U7vei7F5zzQlyTDIdgWNJx3SKgxG21A3BKCQ10V3dhfwm+4XSh3ctbtFzqyWynuCHN24F/qpID0KZdF5pltYlfy+8h5PzJRa+BD+EmYW+iZvk/hJm+uIgZ1OLVl3/tNWg9buG0FaU30bY0fpoff3S/8eqEU7tmo355qhLPXfr36moXyIHqlgvewYp6I84VXQY3Wj5QUfsHXMOc9Y1l2O4Hyx7vED69zT9ISN2Hw11Xm/WBnptROtacreWMvVvtMLcO1HKGhv6Z+yrmunPJv8cbpXQXaCXnsjIXyhp70kNEfm582mb60A7EsOjwcBnHCCNqu5juUpt9IrpyzWVbVfPBFZoDaLRv5MBYHenKIVKyssKFh5F8VU9yLW3HFyX8UadOcBhpIfZwWtp+afU+jNK4cPDI7Mg/IydJJkp0U3AcJMZ2ZsUHRPloR/9r/39T/w9ohxySJ9q4C+zCRfuRPQsXUyjsImnorAlC+q6e990lte6l42g6rrqQsImzW/HhjmTehQLjhj3Q/EHlQ9K2txzUn+l8/bk29JrfjEJ/BGb7WLPPby6KR4i1zlgerT0k0OjwHXqz10IhcryakRxrALEwjPGUWAumbvcw8bXfz1WTB5s5Om53N6MOeTsOEA6uoebjr1k8F8oxHo+Ydo5lx1Vqq/ZII6fNodlpsfHV+d29/RNVJ+jDO3IuWh+zOtk4THixPolx8nIkX3KeeiDm5PCZat2062aVHD+zT4xzq+eXbgyMDYTUV3/HB+7+Onev69PVvdAWG7GLd2m2e7d39c3uDf5oN0ab80YW7PEJC3EJQeecY8DLHK9X6sh1g8KNjHPvydjo9+3eMXh9mAxCUZJn+q1FaW6fgLl+QDaBjRegpjns+dDxLyXbOzCQB8JdPX3y+0an64IJHveOzHnviM58+bjXPQE+w/g7HwZn5zMrwrRFuYxyakDxec9cxBFaMHgFPtuUQC5uvpFibt+LWxwqkwSBPOIMYHqLQ0ENlBncCLihYPylwDs+INVCKjwGS7OMIk1u8JpFvCnQalK4iVo1r2E1VMURWd/3OPr16VhIWsqve0spCPU7OpY5JaFRK3ioHiX6RgB1kvwsdG8k5stx5Jya4zEL5QlwXJt9J0zMoZGDVTA2jamlZ8+37sTAIdch1KGx9qY7noFOsONARZuCPXLo+vuqHl8OvTrommaKGqcZxEOzJmp86Ln4hb7Z8w5ny8462mrn08DzfvnMGDj/C+a11xxy3JKeH/j10O0ySFP7EImlsXwr/rqhn1LBHA/ICnK/60g5x/vYxdQd6RYlEpdgdhexYmJDPiFZFmtJh8XW5+j6o/U53JpvBBzgVX0i1GSLr8n2PL+WUhbbQ12tfddH50OTfW5HfXDI9A/7ik41PeQZcj2ymKrs8wPo1P7MMOmACVdO8auR0+aBj12oTce55sP+znHAL7J/l+uMx3PodHhOY2Cf7sDzJssdL7zvMnzPhWkPFC3zmSzivXsQcDfec+Vd7XawSwcjc3CNQsZfWYWErpA92+5gkIjj7KsJRYThPlbXuZVHFWqYA0COC0qJm1qlHwAdNEC1ZRHWepVCbxwc+M5QFEbKj15nMPsq7B9m8iMzq9BUuVqJKDG1fOrlfpgf/RvvTkYL2zhY2JvCthdspDKT/GYfcGPa6BFxO8Sj4kND7opFPSUf6GwTf/d/NmXFBSzEDRk0OkWEvoY3bb3D+4nDS8xsoxIIoiiC4OLiAgMrht0R/ukO7SWI7rT1wNQOdYBos2jPetKnBMfsLVpWzVHCP44fr2gnplP6PNcJe3gfb3JoEMUbJf+6Ee0Yfz599yys75FlyEGBeP5hDOrMIKe7bdpQ1aGMbH6a5umeL4cdGThj8RtzfyHraicAjo+uY8NQHBNZLvYitkziLgeH9b0dxFFHZ+ncXG0O/ZAKT7Cb0R+eg9X6AQ9T5oYaNWlCP2wLk+CcT1uha9SSDdFAwffCEXaDJOgR2UEzL1KiopYzPD+WeEXZGCd1tN9867y0dzF7DmrdqWkLU1voDtL+B0+qpBnI0uGIcVE3h49dLE5Zw/PTHKY92NpH4xQ6vhC31LiVm2N0CKv1GDtltZvTLTTeS6fMuiKdsv2Kta/xmOByEB5ma4o4Q7jjKSQVRBaRMGkLND3iP4W1YqRkozpoHddPiDfFeazRgrV5K+VSYco3vM2KqRNFvKGr2lyTrpUZYWO8j4hV1pPpREOttvlKNAOk/ovGhXGk5tFIMVs6pwLo6OBM/I2eMoul1Rm8h8ZRqjfL3W7X5tSiiE3OyiaVfrPpzh6Zq2X/puerWOeWdW5ZviocwtwvrvRTORaSCdosVckebl12ALf+soc7FepIDRPKiZzKcK9FV8Ixw67RsVnho89Ah66iM0UUKSUHYzRr2A0ZQaY7cZI6YyPH3pvZREewvBbSO9oDaUq7wR202TrfQ4m/z0A5BSo9YQ9r+zXsBqP8AmmGhHoR4DRfB2JgG/gP0ahjuz9DmY0Qgs7+rCAIGwYEHbPQ7kC4sT54ky5LhEZVTwsbuY15MCu61ppvPn7ERIRrvFb6DPMObjCli/XDcXahOMolZdDT0zqi9KFTde5ziIbSyWxpl8G/nYr/h/lGYVwWD8s5BWZJ/TvoGqUJpyqN3UjJRTV1OpfkBcZTj87OO0EX4E+rLafLiKpHdZBs69PMbxU4RsW3dH+7NW4t03nLb4lbTN2xV1gVm0QUl5jVtWyNJ7NPfgtz/bbGP7TSbFm2xrzZtJCvZfxBFLaYUwkiNDK6NW6x3mv5rUUWb8rWmLCJAttKZJXVNXW5eY03ZZWt5Udb9AkzUqdplJciubQf30VzJLP1NpUJYaVLD/PstjUjeGyg9U+L+8m9QdzyWzgt8AZza0z7S3p/u4qS7MYWoUv5JXtzTcuf/i+/7qx1G6YAAA==)), haven't taken the time to dig into either the cause nor the weird machine-dependent behaviour, but indeed I think that assert shouldn't have been there at the first place.

Reply:
Sounds good; we probably have better things to do than chasing flakes right now :)
I'll make a final read-through and try to break this, but I feel we're very close.